shanoir-ng-front/src/app/studies/study/study.component.html

```diff
@@ -421,7 +421,7 @@ <h2 class="header command-zone" i18n="Create study|Title@@studyDetailCreateTitle
 					(delete) = "delete()"
 					(cancel)="goToView()"
 					(back)="goBack()">
-				<dataset-download  *ngIf="mode == 'view' && hasDownloadRights()" [studyId]="id" [datasetIds]="selectedDatasetIds"></dataset-download>
+				<button class="right-icon dl-button" type="button" (click)="downloadAll()">Download<i class="fas fa-download"></i></button>
```

Comment:
Download button is now available during creation and edition, it should probably not. ( + check rights ?)